### ISSUE TITLE
LMFDB map

### ISSUE THREAD

**Issue by alinabucur** [LMFDB/lmfdb#442]
Make picture with javascript functionality.

**Comment by davidfarmer** [comment]
Alina is working on this

**Comment by jvoight** [comment]
Alina also says: I need (supershort, but informative) descriptions and examples for all the elements in there.  Alina, do you want to create a file where we can edit them in parallel, or do you want to just receive them by e-mail?

**Comment by jvoight** [comment]
Closed as duplicate of #575.
